test/envtest/qatdeviceplugin_controller_test.go

```diff
@@ -46,9 +46,14 @@ var _ = Describe("QatDevicePlugin Controller", func() {
 				Name: "qatdeviceplugin-test",
 			}
 
+			annotations := map[string]string{
+				"container.apparmor.security.beta.kubernetes.io/intel-qat-plugin": "unconfined",
+			}
+
 			toCreate := &devicepluginv1.QatDevicePlugin{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: key.Name,
+					Name:        key.Name,
+					Annotations: annotations,
```

Comment:
test with and without, e.g., by adding them like this but then updating the CR and checking they're not there anymore?

Reply:
done